scripts/create-clusters.sh

```diff
@@ -141,27 +132,23 @@ function create-clusters() {
 function fixup-cluster() {
   local i=${1} # cluster num
 
-  local kubeconfig_path="$(kind get kubeconfig-path --name cluster${i})"
-  export KUBECONFIG="${KUBECONFIG:-}:${kubeconfig_path}"
-
   if [ "$OS" != "Darwin" ];then
     # Set container IP address as kube API endpoint in order for clusters to reach kube API servers in other clusters.
-    kind get kubeconfig --name "cluster${i}" --internal >${kubeconfig_path}
+    OLD_KUBECONFIG=${KUBECONFIG:-}
+    export KUBECONFIG=$(mktemp)
+    kind get kubeconfig --name "cluster${i}" --internal > ${KUBECONFIG}
+    SERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}')
```

Comment:
@makkes should not this be `[${i}]` instead of `[0]` ?

Reply:
@hectorj2f, good question. The `[0]` is actually intentional since `kind get kubeconfig` will get one and only one kubeconfig (for cluster `cluster${i}`, thus it will only contain one entry in the `clusters` array and this is the one that we want to get the server address from. We're doing all of this in the temporary kubeconfig file created two lines above which is deleted as soon as we have the internal server address.